audit-service/src/test/java/uk/gov/gchq/palisade/service/audit/service/LoggerAuditServiceTest.java

```diff
@@ -105,7 +104,7 @@ public void auditRegisterRequestException() {
                 .withResourceId(resource.getId())
                 .withContext(context)
                 .withException(exception)
-                .withServiceClass(Service.class);
+                .withServiceName("Service");
```

Comment:
Use one of the Enums, otherwise people might think that it is acceptable to put "service" in here.

Reply:
`ServiceName` is in `palisade-service`. Should the enum be copied or the enum moved to common?

Reply:
using one of the enums